### PR TITLE
OSSM-1931: created folder to install "acorn"module

### PR DESCRIPTION
**Pull request by dcillera**
OSSM-1931: created node_modules folder in /opt/emdsk to install acorn module

### CODE REVIEW

File: docker/maistra-builder_2.3.Dockerfile [hunk 1]
```diff
@@ -76,6 +76,9 @@ RUN curl -sfL https://download.docker.com/linux/centos/docker-ce.repo -o /etc/yu
                    binaryen emsdk docker-ce rubygems npm yarn rpm-build && \
     dnf -y clean all
 
+RUN mkdir /opt/emsdk/node_modules
```
Comment (jwendell) [inline docker/maistra-builder_2.3.Dockerfile:79]: Move this to the end of the file with the others mkdirs and chmods

Comment (dcillera) [reply docker/maistra-builder_2.3.Dockerfile:79]: ok

Comment (jwendell) [reply docker/maistra-builder_2.3.Dockerfile:79]: and use mkdir -p

Comment (dcillera) [reply docker/maistra-builder_2.3.Dockerfile:79]: ... and use: && \

**Comment by jwendell** [comment]
Closing in favor of another approcah

**Pull request closed without merging**